spring-redisgraph-example/src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 spring.redisgraph.host=localhost
-spring.redisgraph.port=6379
\ No newline at end of file
+spring.redisgraph.port=6379
+spring.redisgraph.password=Redis@password123
```

Comment:
please comment this line and add the explanation above, for the usage of this field.
This is an example application configuration, we assume that in most cases Redis server default settings are applied.
Also, our circle ci container runs a server with default settings, we do not want it to fail due to configurations.

Reply:
Commented. No such default password, No password is set as default